docs/src/pages/playground.tsx

```diff
@@ -0,0 +1,99 @@
+import clsx from 'clsx';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import Layout from '@theme/Layout';
+
+import styles from './playground.module.css';
+import { useState } from 'react';
+import Options from '../components/PlaygroundRenderSettings';
+import ImageDisplay from '../components/PlaygroundPreview';
+import CodeInputArea from '../components/PlaygroundCodeEditor';
+import toast, { Toaster } from 'react-hot-toast';
+
+const backendUrl = 'http://localhost:8081/render_image';
```

Comment:
- const should be written with all upper case `BACKEND_URL`
- actual backend url is `http://localhost:8081` this name is misleading
- I would write some helper that returns url for path (especially that we will need to change URL depending on environment)